src/devices/Card/Mifare/AccessType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Card.Mifare
+{
+    /// <summary>
+    /// The type of access for the data sectors
+    /// </summary>
+    [Flags]
+    public enum AccessType
+    {        
+        Never = 0b0000_0000,
```

Comment:
Never or None? (`Never access type` just doesn't sound right)

Reply:
OK, will change for None

Reply:
Changed as well in the AccessSector